src/bundle/Resources/views/themes/admin/trash/list.html.twig

```diff
@@ -232,7 +232,7 @@
                 },
             ]) %}
 
-            {% set has_search_params = trash_search_params ? trash_search_params.creator or trash_search_params.content_type or trash_search_params.trashed or trash_search_params.section : false %}
+            {% set has_search_params = trash_search_params|length > 0 %}
```

Comment:
I think we cannot simplify it like this. This flag is responsible for the header of the table, it should always be `List (x)` unless you filter something. The second page is not a "filter" and I believe with this solution on the second page the header will be `Resaults (x)`